common/datasets/librispeech.py

```diff
@@ -410,18 +410,21 @@ def get_g2p_augmented_bliss_lexicon_dict(use_stress_marker=False, subdir_prefix=
     original_bliss_lexicon = get_bliss_lexicon(
         use_stress_marker=use_stress_marker, subdir_prefix=subdir_prefix
     )
-    g2p_augmenter = G2PBasedOovAugmenter(original_bliss_lexicon=original_bliss_lexicon)
+    current_bliss_lexicon = original_bliss_lexicon
 
     bliss_corpus_dict = get_bliss_corpus_dict(subdir_prefix=subdir_prefix)
-    for corpus_name, bliss_corpus in bliss_corpus_dict.items():
+    for corpus_name, bliss_corpus in sorted(bliss_corpus_dict.items()):
         if "train" in corpus_name:
-            augmented_bliss_lexica[
-                corpus_name
-            ] = g2p_augmenter.get_g2p_augmented_bliss_lexicon(
+            g2p_augmenter = G2PBasedOovAugmenter(
+                original_bliss_lexicon=updated_bliss_lexicon,
+                train_lexicon=original_bliss_lexicon,
+            )
+            current_bliss_lexicon = g2p_augmenter.get_g2p_augmented_bliss_lexicon(
```

Comment:
the update step will be empty for "460" and "960" then, because we already have all OOVs in the earlier lexicas, so those two can be skipped (lex460 = lex360 and lex960 = lex500)

Reply:
This is true, but what do you propose to change? Should there be exceptions like `if corpus_name=='train-clean-460': <copy 360>` or should I hard code the three relevant corpora again?

Just asking to find the most elegant solution.

Reply:
```python3
if corpus_name in ['train-clean-460', 'train-other-960']:
    augmented_bliss_lexica[corpus_name] = current_bliss_lexicon
else:
    do stuff....
```

Reply:
So in those cases the current one is already the "correct" one, or not?